cmd/root.go

```diff
@@ -75,15 +78,15 @@ func init() {
 	importCmd.Flags().StringVar(&importRelease, "release_name", "", "The name of the release to import.")
 	importCmd.Flags().StringVar(&importRepository, "repository", "", "The helm repository for the imported release.")
 
-	// add commands here
-	rootCmd.AddCommand(plotCmd)
-	rootCmd.AddCommand(convertCmd)
-	rootCmd.AddCommand(templateCmd)
-	rootCmd.AddCommand(diffCmd)
-	rootCmd.AddCommand(lintCmd)
-	rootCmd.AddCommand(getManifestsCmd)
-	rootCmd.AddCommand(updateCmd)
-	rootCmd.AddCommand(importCmd)
+	rootCmd.AddCommand(plotCmd,
```

Comment:
nit: `plotCmd,` could go on the next line so it looks prettier

Reply:
will do, not sure why I didn't originally